library/std/src/time.rs

```diff
@@ -458,11 +458,12 @@ impl SystemTime {
     /// # Examples
     ///
     /// ```
-    /// use std::time::SystemTime;
+    /// use std::time::{Duration, SystemTime};
     ///
     /// let sys_time = SystemTime::now();
-    /// let difference = sys_time.duration_since(sys_time)
-    ///                          .expect("Clock may have gone backwards");
+    /// let new_sys_time = sys_time.checked_add(Duration::new(1, 0)).unwrap();
+    /// let difference = new_sys_time.duration_since(sys_time)
+    /// .expect("Clock may have gone backwards");
```

Comment:
```suggestion
    ///     .expect("Clock may have gone backwards");
```
Oh, not like this, this is too under-indented which is not so good for readability.